docs/reference/indices/rollover-index.asciidoc

```diff
@@ -4,8 +4,11 @@
 <titleabbrev>Rollover index</titleabbrev>
 ++++
 
-Assigns an <<indices-aliases, index alias>> to a new index
-when the alias's existing index meets a condition you provide.
+Creates a new index for a rollover target when the target's existing index meets a
```

Comment:
👍  I feel like 'rollover target' is a good term for this and it's precisely explained here.